roadmap/implementers-guide/src/node/backing/statement-distribution.md

```diff
@@ -1,107 +1,453 @@
+-   `Statement` messages contain only a signed compact statement, without full
+    candidate info.
+-   `BackedCandidateManifest` messages advertise a description of a backed
+    candidate and stored statements.
+-   `BackedCandidateAcknowledgement` messages acknowledge that a backed candidate
+    is fully known.
+
+### Request/response protocol
+
+Nodes can request the full `CommittedCandidateReceipt` and
+`PersistedValidationData`, along with statements, over a request/response
+protocol. This is the `AttestedCandidateRequest`; the response is
+`AttestedCandidateResponse`.
+
+### Importability and the Hypothetical Frontier
+
+-   The ****prospective parachains**** subsystem maintains prospective "fragment
+    trees" which can be used to determine whether a particular parachain candidate
+    could possibly be included in the future. Candidates which either are within a
+    fragment tree or <span class="underline">would be</span> part of a fragment tree if accepted are said to be
+    in the "hypothetical frontier".
+-   The ****statement-distribution**** subsystem keeps track of all candidates, and
+    updates its knowledge of the hypothetical frontier based on events such as new
+    relay parents, new confirmed candidates, and newly backed candidates.
+-   We only consider statements as "importable" when the corresponding candidate
+    is part of the hypothetical frontier, and only send "importable" statements to
+    the backing subsystem itself.
+
+### Cluster Mode
+
+-   Validator nodes are partitioned into groups (with some exceptions), and
+    validators within a group at a relay-parent can send each other `Statement`
+    messages for any candidates within that group and based on that relay-parent.
+    This is referred to as the "cluster" mode.
+-   `Seconded` statements must be sent before `Valid` statements.
+-   `Seconded` statements may only be sent to other members of the group when the
+    candidate is fully known by the validator.
+-   Validators in a cluster receiving messages about unknown candidates request
+    the candidate (and statements) from other cluster members which have it.
+-   Spam considerations
+    -   The maximum depth of candidates allowed in asynchronous backing determines the
+        maximum amount of `Seconded` statements originating from a validator V which
+        each validator in a cluster may send to others. This bounds the number of
+        candidates.
+    -   There is a small number of validators in each group, which further limits
+        the amount of candidates.
+
+&#x2014;
+
+-   We accept candidates which don't fit in the fragment trees of any relay
+    parents.
+    -   We listen to prospective parachains subsystem to learn of new additions to
+        the fragment trees.
+    -   Use this to attempt to import the candidate later.
+
+### Grid Mode
+
+-   Every consensus session provides randomness and a fixed validator set, which
+    is used to build a redundant grid topology.
```

Comment:
Why is it redundant?

Reply:
It's redundant in the sense that there are 2 paths from every node to every other node